pkg/kp/rcstore/fake_store.go

```diff
@@ -72,6 +73,25 @@ func (s *fakeStore) List() ([]fields.RC, error) {
 	return results, nil
 }
 
+func (s *fakeStore) WatchNew(quit <-chan struct{}) (<-chan []fields.RC, <-chan error) {
+	go func() {
```

Comment:
yeah you're right, will fix. I wasn't sure what the fake's watchnew should look like but it probably doesn't matter